specification/app/resource-manager/Microsoft.App/stable/2022-05-01/ManagedEnvironments.json

```diff
@@ -714,6 +714,59 @@
         }
       }
     },
+    "CustomDomainConfiguration": {
+      "type": "object",
+      "description": "Configuration properties for apps environment custom domain",
+      "properties": {
+        "customDomainVerificationId": {
+          "description": "Id used to verify domain name ownership",
+          "type": "string",
+          "readOnly": true
+        },
+        "dnsSuffix": {
+          "type": "string",
+          "description": "Dns suffix for the environment domain",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "certificateValue": {
+          "format": "byte",
+          "description": "PFX or PEM blob",
+          "type": "string",
+          "x-ms-mutability": [
```

Comment:
with x-ms-secret, x-ms-mutability is not necessary. @leni-msft  for confirmation.

Reply:
yes, you're right. See [here](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret) :
> "x-ms-secret": true is equivalent to "x-ms-mutability": ["create", "update"].

Reply:
Thanks, updated